common/src/sql/dbinit.sql

```diff
@@ -1105,7 +1105,13 @@ CREATE TYPE omicron.public.ip_kind AS ENUM (
      * known address that can be moved between instances. Its lifetime is not
      * fixed to any instance.
      */
-    'floating'
+    'floating',
+
+    /*
+     * A service IP is an IP address not attached to a project nor an instance.
+     * It's intended to be used for internal services.
+     */
+    'service'
 );
 
 /*
```

Comment:
This comment is actually intended to be attached to the `CREATE TABLE` statement, GitHub just apparently doesn't allow comments on lines that are not on or near changes in the actual PR.

At some point, doesn't have to be now, we probably want to rename this table to just `external_ip`. It's no longer just associated with addresses for guests.

Reply:
I'll do that in a follow-up PR to this change -- hopefully that keeps the "simple change" piece of renaming separate from this PR, which does have some more subtle changes.

https://github.com/oxidecomputer/omicron/issues/1614